### PR TITLE
Added SP

### PR DESCRIPTION
**Pull request by Himani-ja**

### CODE REVIEW

File: 05/PetStore/PetStore/Program.cs [hunk 1]
```diff
@@ -12,25 +12,30 @@ class Program
...
                 .AddJsonFile("appSettings.json")
                 .Build();
 
             string conString = config.GetConnectionString("PetDb");
             GetCatsDisconnected(conString);
-           /*
+            
+             GetCats(conString);
+
+             Console.Write("PLease enter the Id of the cat whose name is to be changed ");
```
Comment (kaurrevature) [inline 05/PetStore/PetStore/Program.cs:26]: refactor this code to a method

File: 05/PetStore/PetTest/PetTest.csproj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
```
Comment (kaurrevature) [inline 05/PetStore/PetTest/PetTest.csproj:3]: don't push this file

**Review by kaurrevature — CHANGES_REQUESTED** [review]

File: 05/PetStore/PetStore/Program.cs (hunk 1 above)
Comment (Himani-ja) [reply 05/PetStore/PetStore/Program.cs:26]: ma'am this code is not ours

**Comment by Himani-ja** [comment]
Updating code

**Pull request closed without merging**